egs2/hifitts/tts1/conf/tuning/train_full_band_multi_spk_vits.yaml

```diff
@@ -0,0 +1,187 @@
+# This configuration is for ESPnet2 to train 44.1 kHz
```

Comment:
Please only PR the config related to this recipe (usually start with a single best config would be sufficient)